cmd/up/space/prerequisites/manager.go

```diff
@@ -55,42 +56,56 @@ type Status struct {
 }
 
 // New constructs a new Manager for working with installation Prerequisites.
-func New(config *rest.Config, defs *defaults.CloudConfig, features *feature.Flags) (*Manager, error) {
+func New(config *rest.Config, defs *defaults.CloudConfig, features *feature.Flags, versionStr string) (*Manager, error) { // nolint:gocyclo
 	prereqs := []Prerequisite{}
-	certmanager, err := certmanager.New(config)
+
+	version, err := semver.NewVersion(versionStr)
 	if err != nil {
-		return nil, errors.Wrap(err, errCreatePrerequisite)
+		return nil, errors.New("invalid version format: " + err.Error())
 	}
-	prereqs = append(prereqs, certmanager)
 
-	uxp, err := uxp.New(config)
+	constraint, err := semver.NewConstraint("< v1.7.0-0")
```

Comment:
nit for readability:

```suggestion
	requiresUXP, err := semver.NewConstraint("< v1.7.0-0")
```

and below

```go
if requiresUXP.Check(version) {
  ...
```